networkx/algorithms/community/asyn_lpa.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+#    Copyright (C) 2015
+#    All rights reserved.
+#    BSD license.
+
+import random
+import networkx as nx
```

Comment:
Imports should be organized as follows: built-in libraries first, third-party libraries second, current library (networkx) third.